server/DB/config/connections.js

```diff
@@ -0,0 +1,28 @@
+const {pool, Pool} = require('pg');
+require('env2')('./config.env');
```

Comment:
I think you are going to use the environment variables in other places than the connection file like in auth controllers for `SECRET_KEY, so you should require the `env` in the `app.js` maybe